.github/workflows/vapoursynth.yml

```diff
@@ -0,0 +1,151 @@
+name: vapoursynth
+
+on: [push, pull_request]
+
+jobs:
+  clippy-rustfmt:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install stable
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: stable
+        override: true
+        components: clippy, rustfmt
+    - name: Run clippy
+      uses: actions-rs/clippy-check@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        args: --all --all-features
+    - name: Run rustfmt
+      uses: actions-rs/cargo@v1
+      with:
+        command: fmt
+        args: --all -- --check
+
+  unix-tests:
+    strategy:
+      matrix:
+        toolchain:
+         - stable-x86_64-apple-darwin
+         - stable-x86_64-unknown-linux-gnu
+         - stable-i686-unknown-linux-gnu
```

Comment:
Would it be possible to include beta and nightly targets, too? It's customary in Rust to test on all three, and set nightly as allowed failure.

Reply:
Sorry, but I'm not going to add them. I think the code should be tested with the stable version only. The same goes for the CI system too. It's a personal choice of mine. Feel free to add them after this PR is landed though, no problem on my part :)